packages/core-data/src/test/selectors.js

```diff
@@ -20,43 +21,47 @@ import {
 	getReferenceByDistinctEdits,
 } from '../selectors';
 
-describe( 'getEntityRecord', () => {
-	it( 'should return undefined for unknown record’s key', () => {
-		const state = deepFreeze( {
-			entities: {
-				data: {
-					root: {
-						postType: {
-							queriedData: {
-								items: {},
-								queries: {},
+// getEntityRecord and getEntityRecordNoResolver selectors share the same tests
+Object.entries( { getEntityRecord, getEntityRecordNoResolver } ).forEach( ( [ name, func ] ) => {
```

Comment:
Can we use `it.each`?

https://jestjs.io/docs/en/api#2--testeachtablename-fn-timeout-

Reply:
Wow I love this, didn't know about it.